src/modules/sdk/objections/service.ts

```diff
@@ -121,6 +122,12 @@ export const deleteAttachment = async (companyNumber: string,
   return;
 };
 
+export const downloadAttachment = async (downloadUri: string,
+                                         httpResponse: Response,
+                                         token: string) => {
+  return Promise.resolve();
```

Comment:
I'd suggest adding a TODO comment that references the sub-task which describes replacing this stub with the actual implementation.

Reply:
updated